manifests/agent.pp

```diff
@@ -281,11 +284,22 @@
     }
   }
 
-  if $facts['kernel'] == 'windows' and $manage_choco {
-    package { $zabbix_package_agent:
-      ensure   => $zabbix_version,
-      provider => $zabbix_package_provider,
-      tag      => 'zabbix',
+  if $facts['kernel'] == 'windows' {
+    if $manage_choco {
+      package { $zabbix_package_agent:
+        ensure   => $zabbix_version,
+        provider => $zabbix_package_provider,
+        tag      => 'zabbix',
+      }
+    } else {
+      assert_type(Stdlib::Windowspath, $zabbix_package_source)
```

Comment:
That's not needed since the type is asserted above

Reply:
I thought so, but i was specifically told to add the `assert_type()` while i was in #voxpupuli. I'll remove it then.

Reply:
the `assert_type()` here is required because it enforces Stdlib::Windowspath while being on Windows. Above it's `Optional[]`. Please leave it in.

Reply:
I'll leave it then